README.md

```diff
@@ -179,7 +179,7 @@ To run, build and test locally:
     * Run the following commands in the root directory:  `[root directory] $ npm install` and `[root directory] $ npm run build-theme`
         * this will compile the theme sass, copy the static assets to the theme `resource` directory and compile the typescript for WebAuthn.
 4. Build both spi jars:
-    * From the root directory run the following command: `sbt govUkNotifySpi/assembly eventPublisherSpi/assembly`
+    * From the root directory run the following command: `sbt assembly`
```

Comment:
Not sure these instructions here: https://github.com/nationalarchives/tdr-auth-server/blob/master/README.md#update-tdr-theme-locally will still work now the `standalone-ha.xml` is gone?

I think might need to go through in the whole `README` to check it.

Reply:
Some of that README references stuff that hasn't been true for a year or so. I've gone through and tried to update it where it's wrong.